newrelic.js

```diff
@@ -11,5 +11,13 @@ exports.config = {
   app_name: ['cp-events-service'],
   agent_enabled: false, // set via NEW_RELIC_ENABLED for production
   license_key: '',      // set via NEW_RELIC_LICENSE_KEY
-  filepath: '/tmp/newrelic_agent_events.log'
+  filepath: '/tmp/newrelic_agent_events.log',
+  logging: {
+    /**
+     * Level at which to log. 'trace' is most useful to New Relic when diagnosing
+     * issues with the agent, 'info' and higher will impose the least overhead on
+     * production applications.
+     */
```

Comment:
Please add the obfuscated query too :)
https://github.com/CoderDojo/cp-dojos-service/commit/8059a9286846b9b51ab4abef59e406275f4165bc#diff-b7d1225dd38f3342e35ccf6f63f71a69R15